resolwe/flow/executors/__init__.py

```diff
@@ -55,7 +55,7 @@ def run(self, data_id, script, verbosity=1):
         if verbosity >= 1:
             print('RUN: {} {}'.format(data_id, script))
 
-        self.data_id = data_id
+        self.data_id = str(data_id)
```

Comment:
We might wish to compare somewhere, but the id there might be in original form (int) that would not match the string here.